assets/src/edit-story/elements/video/frame.js

```diff
@@ -14,14 +14,127 @@
  * limitations under the License.
  */
 
+/**
+ * External dependencies
+ */
+import { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { CSSTransition } from 'react-transition-group';
+
 /**
  * Internal dependencies
  */
 import StoryPropTypes from '../../types';
 import MediaFrame from '../media/frame';
+import { elementFillContent } from '../shared';
+import { useStory } from '../../app/story';
+import { ReactComponent as PlayIcon } from '../../icons/play.svg';
+import { ReactComponent as PauseIcon } from '../../icons/pause.svg';
+
+const Play = styled(PlayIcon)`
+  width: 48px;
+  height: 48px;
+`;
+const Pause = styled(PauseIcon)`
+  width: 48px;
+  height: 48px;
+`;
+
+const Wrapper = styled.div`
+  ${elementFillContent}
+`;
+
+const ButtonWrapper = styled.div`
```

Comment:
This should really be a button tag and not a div.

Reply:
Then it will steal the TAB, so not sure about it, I agree that this video element should play/pause on SPACE, so then button here doesn't make sense? I think we should first think about the keyboard interaction with elements on the canvas, globally, how ENTER/SPACE should work etc.

Reply:
This is clearly a button. The user clicks and something happens. That is the definition of a button. Screen readers treat button tags differently than divs. Either it is a button or you add `role="button"`

Reply:
Hi everyone, the play icon has to be a button because there is a case where if you click outside of the video then hover over it again you see the media controls (play button, scrubber). You can 1) click on the button to play the video or click anywhere in the video but not the play button to just select the video so that you can bring up the video attributes in the design panel. Let me know if this helped answer the question.

Reply:
@samitron7 it works as you described (it is a clickable element), we are just talking about the semantics/a11y.

Not sure if `<div role="button">` without `tabindex` is correct and my main focus is on keyboard navigation and focus stealing. Do we have a document describing such things? (how to operate canvas elements with keyboard, @samitron7 ?)

Reply:
I think `<role=button>` without `tabindex` is almost always not correct. But it might suffice to set `tabindex` to `-1` for this particular case. SPACE on the frame to play/pause might be just fine.